### PR TITLE
add a new QC filter

### PR DESCRIPTION
**Pull request by guoqing-noaa**
## Description
Added a new QC filter

## Definition of Done
Unit Test passed

### Issue(s) addressed
add a new QC filter

## Dependencies
none

## Impact
- [ ] ufo

### CODE REVIEW

File: src/ufo/filters/PracticalBoundsCheck.cc [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+	// Sanity checks
+	  if (filtervars.nvars() == 0) {
+	    oops::Log::error() << "No variables will be filtered out in filter "
+			       << config_ << std::endl;
+	    ABORT("No variables specified to be filtered out in filter");
+	  }
+
+	// Loop over all variables to filter
+	    for (size_t jv = 0; jv < testvars.nvars(); ++jv) {
+	      //  get test data for this variable
```
Comment (cshartsough) [inline src/ufo/filters/PracticalBoundsCheck.cc:61]: trying out sample comment. :-)

**Review by cshartsough — APPROVED** [review]
looks good.  I approve.

**Review by imoradi — COMMENTED** [review]
No comment

**Review by christinaholtNOAA — APPROVED** [review]

**Comment by LiaofanLin-NOAA** [comment]
looks good to me.

**Review by LiaofanLin-NOAA — COMMENTED** [review]
Looks good to me!

**Comment by imoradi** [comment]
I don’t see this in review request though am getting the emails.

**Comment by guoqing-noaa** [comment]
Sorry that I played further and already merged this PR to the "develop" branch. It is in the "1 closed" (just next to "7 open"). If you click "1 closed", you will see this PR.

**Pull request merged**